### DISCUSSION TITLE
Relatórios em pdf.

### DISCUSSION THREAD

**Discussion by NetoPCGO** [sepinf-inc/IPED#2321] [Q&A]
Quando geramos relatório, poderíamos gerar um arquivo único em pdf? E as conversas html ficam individuais, teria um meio de unificarmos elas num arquivo só, html, ou mesmo pedf? (Em se tratando de Whatsapp)

**Comment by wladimirleite** [comment]
Hello @NetoPCGO!
There is an open issue for this: https://github.com/sepinf-inc/IPED/issues/1039.

**Reply by NetoPCGO** [reply]
Muito obrigado!
